### PR TITLE
Adds SYNCS logo to header, and match header styling to website

### PR DESCRIPTION
**Pull request by AndrewOCC**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
     <link rel="stylesheet" href="static/bootstrap.min.css" />
     <link rel="stylesheet" href="static/custom.css" />
   </head>
   <body>
-    <nav class="navbar navbar-dark bg-primary">
-      <a class="navbar-brand" href="https://syncs.org.au">SYNCS</a>
+    <nav class="navbar navbar-dark bg-syncs">
+      <div class="container">
+        <div class="logo">
+          <h1><a href="/"><img src="static/logo-blue.svg" alt="SYNCS" height="32" /></a></h1>
```
Comment (auscompgeek) [inline index.html]: Any particular reason for the h1?

Comment (AndrewOCC) [reply index.html]: I believe it's the same as the website, just meant the styles could align a bit easier

Comment (auscompgeek) [reply index.html]: I think the website CSS actually does target h1, which is why I left it like that. I don't see a good reason to do it here given it's not necessary and doesn't make sense semantically either.

Also prettier isn't happy with the HTML as written either. (Turns out PrettierCI doesn't check HTML files by default.)

Comment (AndrewOCC) [reply index.html]: Ahh my work laptop runs prettier on save, mustn't have set that up on home one 😖

**Pull request merged**